app/public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Express App</title>
+</head>
+<body>
+    <h1>Client</h1>
+    <button id="getHealthButton">Get health</button>
+    <script src="/client.js"></script>
```

Comment:
That script doesn't impact the render of the page and is not a performance issue, so it should be put in head tags and not inside the body.